source/options.rst

```diff
@@ -84,6 +93,80 @@ To view the created configmap, use the following command:
 
    kubectl describe configmaps cluster1-pxc
 
+Use a Secret Object
+-------------------
+
+The Operator can also store configuration options in `Kubernetes Secrets <https://kubernetes.io/docs/concepts/configuration/secret/>`_.
+This can be useful if you need additional protection for some sensitive data.
+
+You should create a Secret object with a specific name, composed of your cluster
+name and a specific suffix:
+
+* ``my-cluster-name-pxc`` for the Percona XtraDB Cluster Pods,
+* ``my-cluster-name-proxysql`` for the ProxySQL Pods,
+* ``my-cluster-name-haproxy`` for the HAProxy Pods,
+  
+.. note:: To find the cluster name, you can use the following command:
+
+   .. code:: bash
+
+      $ kubectl get pxc
+
+Configuration options should be put inside a specific key:
+
+* ``data.pxc`` key for Percona XtraDB Cluster Pods,
+* ``data.proxysql`` key for ProxySQL Pods, 
+* ``data.haproxy`` key for HAProxy Pods.
+
+Actual options should be encoded with `Base64 <https://en.wikipedia.org/wiki/Base64>`_.
+
+For example, let's define a ``my.cnf`` configuration file and put there a pair
+of MySQL options we used in the previous example:
+
+.. code:: yaml
+
+   [mysqld]
+   wsrep_debug=ON
```

Comment:
PXC 8.0 doesn't support ON/OFF for this variable so I would use the new values instead: https://github.com/percona/percona-xtradb-cluster-operator/blob/main/deploy/cr.yaml#L57

More info: https://jira.percona.com/browse/PXC-3034